test/users.test.js

```diff
@@ -59,6 +59,11 @@ describe('Users Methods', () => {
       })
     })
   })
+  it('setDescription() sets a description', async () => {
+    return setDescription("Hello!").then(res => {
+      return expect(res.description).toEqual(expect.any(String));
```

Comment:
We likely want a second test-case with this that is using `getBlurb()` to validate the change actually took place on the website (doing two set requests to watch the change).

On that note; we might also want to consider marking `getBlurb()` as deprecated and aliasing it under `getDescription()` for consistency sake

Reply:
getBlurb is already deprecated, with moving to getPlayerInfo recommended instead.
getPlayerInfo returns description with a "blurb" key, and I now considering it further setDescription could cause confusion with setting group description, though it's fairly obvious from typings & comments.
Could go for `setUserDescription` & add a key to getPlayerInfo that includes description in description instead of blurb (but leaving blurb for backwards compatibility)